### DISCUSSION TITLE
python -m jcvi.compara.catalog ortholog found 0 anchors although last is installed

### DISCUSSION THREAD

**Discussion by VasyaSokolov** [tanghaibao/jcvi#731] [Q&A] [answered]
Hello! I run "python -m jcvi.compara.catalog ortholog --cpus 30 --full rat vulpes --no_strip_names" on the rat and fox genomes. The algorithm worked and did not give a result, writing: «A total of 0 anchor was found. Aborted.».  
I have Last installed and added to the path, by name lasdb and lastal are called from the console, in addition, when running "python -m jcvi.compara.catalog ortholog", if you look at the "top" then lastal is working. If I run lastdb and lastal separately on the rat.bed and fox.bed, I get a normal MAF file.
The program writes that the file is a rat.bed because there is no chromosome NC_051352.1, but if you just do "grep NC_051352.1 rat.bed" then it finds many lines with NC_051352.1 and the format seems correct.
I can't understand why the analysis doesn't work... Any help would be important to me, thank you!
![обращение к lastal](https://github.com/user-attachments/assets/7121d7ac-8ef9-4a1f-a3b9-2e30df453ec7)
![результат_Mcscan](https://github.com/user-attachments/assets/07632e17-d8da-41bd-9d07-9695188bcb00)
![Ошибка_Mcscan](https://github.com/user-attachments/assets/a3cc399e-9364-447d-97c0-aef78396a22f)

**Comment by tanghaibao** [comment, marked as answer]
@VasyaSokolov 

`jcvi` works on the anchors (genes), not on chromosome sequences. Specifically, it looks at the 4th column in `rat.bed` for `NC_051352.1` where it couldn't find it.

Try instead downloading the gene sequences for rat and then pass it to the command. Make sure the names for the genes in the FASTA header (hence BLAST output) matches the 4th column in the BED file.

**Reply by VasyaSokolov** [reply]
Thank you! I downloaded the .fna files for RNA for rat and fox. I'm trying to launch.

**Reply by VasyaSokolov** [reply]
It works! Thank you!
